cypress/page/address-step.page.ts

```diff
@@ -3,7 +3,7 @@ class AddressStepPage {
   private AddressCheckout: string;
 
   constructor() {
-    this.AddressCheckout = "p.cart_navigation button";
+    this.AddressCheckout = "p.cart_navigation button[name='processAddress']";
```

Comment:
probably, just with `button[name='processAddress']` or `[name='processAddress']` should be enough

Reply:
Thanks Giovanny, its done !!